dffml/util/python.py

```diff
@@ -175,3 +175,27 @@ def within_method(obj: object, method_name: str, max_depth: int = -1) -> bool:
         ):
             return True
     return False
+
+
+def no_inplace_append(list_a: list, arg_b: Union[str, list]) -> list:
```

Comment:
Isn't this the same as `list_a + [arg_b]`?

```python
>>> a = [1]
>>> id(a)
140385771636928
>>> b = a + [2]
>>> id(b)
140385748642624
```

Reply:
True lol :facepalm: